### DISCUSSION TITLE
looks like version 1 of Vector source/sink is still supported

### DISCUSSION THREAD

**Discussion by tshepang** [vectordotdev/vector#11399] [Q&A] [answered]
Looking at https://vector.dev/highlights/2021-08-24-vector-source-sink, it is said there would be no need to set version on Vector source/sink by now, but I see Vector still supports such. Is it an oversight?

**Comment by jszwedko** [comment, marked as answer]
Hi @tshepang ! Yeah, that highlight was a bit overly optimistic with the version numbers. I'll update it. We do expect to drop support for version 1 of the source/sink in 0.22.

**Reply by jszwedko** [reply]
Updated in https://github.com/vectordotdev/vector/pull/11400
